view/template/mail/_settingsForm.php

```diff
@@ -7,12 +7,19 @@
   Javascript is required to securely send your unsubscribe information. Email <a href="mailto:help@lbry.io" class="link-primary">help@lbry.io</a> for manual unsubscription.
 </noscript>
 <form id="<?php echo $formId ?>" novalidate style="display: none">
-  <?php if ($error): ?>
-    <div class="notice notice-error spacer1"><?php echo $error ?></div>
-  <?php endif ?>
-  <div class="notice notice-success hide">Your email preferences have been updated.</div>
+  <div style="min-height: 48px">
```

Comment:
I made the form auto-save. This is an awful design from basically every standard (code quality, maintainability, UX), but otherwise the form would jump on save and the only other alternative was to bite bullet and add snack bars or equivalent UI element.